src/main/webapp/app/app.module.ts

```diff
@@ -40,7 +41,15 @@ import { JhiMainComponent, NavbarComponent, FooterComponent, PageRibbonComponent
         // jhipster-needle-angular-add-module JHipster will add new module here
         SnorlaxEntityModule
     ],
-    declarations: [JhiMainComponent, NavbarComponent, ErrorComponent, PageRibbonComponent, ActiveMenuDirective, FooterComponent],
+    declarations: [
+        JhiMainComponent,
+        NavbarComponent,
+        ErrorComponent,
+        PageRibbonComponent,
+        ActiveMenuDirective,
+        FooterComponent,
+        PoolListComponent
```

Comment:
Why there is no separate `.module.ts` file that could be plugged to imports?

Just like in entities.

Also routing would be in the scope of the module rather than a global list

Reply:
good point `pool` can be a module with all logic concerning pools.